packages/twenty-front/src/modules/object-record/cache/utils/getRecordsFromRecordConnection.ts

```diff
@@ -2,10 +2,17 @@ import { getRecordFromRecordNode } from '@/object-record/cache/utils/getRecordFr
 import { RecordGqlConnection } from '@/object-record/graphql/types/RecordGqlConnection';
 import { ObjectRecord } from '@/object-record/types/ObjectRecord';
 
+type RecordGqlConnectionWithOptionalPageInfo = Omit<
```

Comment:
Do we need to do this change? I'm assuming you don't handle pagination yet but you are still returning a pageInfo right?

Reply:
Mm i remember struggling a bit with this but now it does not seem needed indeed. thanks